src/test/java/org/folio/entitlement/integration/keycloak/KeycloakServiceTest.java

```diff
@@ -355,11 +370,10 @@ void negative_failedToCreateResource() {
       var resource = resource(null, "/r1", scopeWithId("GET"));
       mockResourceCreation(resource, 502, resp -> when(resp.getStatusInfo()).thenReturn(BAD_GATEWAY));
 
-      assertThatThrownBy(() -> keycloakService.updateAuthResources(null, currentDescriptor, TENANT_NAME))
-        .isInstanceOf(IntegrationException.class)
-        .hasMessage("Failed to update authorization resources in Keycloak")
-        .satisfies(error -> assertThat(((IntegrationException) error).getErrors()).containsExactly(
-          parameter("resource", "Failed to create resource: /r1 (status: 502, info: Bad Gateway)")));
+      assertThatThrownBy(() -> keycloakService.updateAuthResources(null, currentDescriptor, TENANT_NAME)).isInstanceOf(
+        IntegrationException.class).hasMessage("Failed to update authorization resources in Keycloak").satisfies(
+          error -> assertThat(((IntegrationException) error).getErrors()).containsExactly(parameter("resource",
+            "Failed to create resource: /r1 (cause: WebApplicationException, causeMessage: HTTP 502 Bad Gateway)")));
```

Comment:
I am not sure what was wrong with the existing formatting, please don't change it if not necessary

Reply:
I've formatted the whole file with our common formatter. If this formatting is somehow wrong - the common formatter should be fixed.